modules/gitops-accessing-the-gitops-operator-metrics.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-the-gitops-operator-performance.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-the-gitops-operator-metrics_{context}"]
+= Accessing the {gitops-shortname} Operator metrics
+
+You can access the Operator metrics from the *Administrator* perspective of the {OCP} web console to track the performance of the Operator.
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default `openshift-gitops-operator` namespace.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Observe* -> *Metrics*.
+
+. Enter the metric in the *Expression* field. You can choose from the following metrics:
+
+* `active_argocd_instances_total`
+* `active_argocd_instances_by_phase`
+* `active_argocd_instance_reconciliation_count`
```

Comment:
cc: @jaideepr97
Also question from Sri: Is this a UI bug? If it is, then when the docs team creates a bug for this, who is the person to whom we can assign this? I will create a bug.

Reply:
This is not a bug. The behavior cannot be consistent because the above metrics are for group of instances whereas this metric is for a single Argo CD Instance. 

If you would like to create a bug to discuss further. Please create the bug on the GitOps board and keep it unassigned.

Reply:
Thanks, @iam-veeramalla for the clarification. No need to create a big then.